source/templates.tex

```diff
@@ -304,9 +304,8 @@
 called \tcode{T}, rather than an unnamed non-type
 \grammarterm{template-parameter} of class \tcode{T}.
 \end{example}
-A storage class shall not be specified in a
-\grammarterm{template-parameter}
-declaration.
+A \grammarterm{storage-class-specifier} shall not appear in a
+\grammarterm{template-parameter} declaration.
```

Comment:
I would marginally prefer to reverse this: "A *template-parameter* declaration shall not have a *storage-class-specifier*."